### ISSUE TITLE
Encryption: correctly include subfolders

### ISSUE THREAD

**Issue by japm48** [TheLocehiliosan/yadm#81]
`yadm encrypt` will fail if the `encrypt` file contains folders or patterns that evaluate to folders.

`ls` on them will list the directory contents, not the full path.

To reproduce:
```bash
mkdir -p ~/mysecrets/a/
echo something > ~/mysecrets/a/sshkey
echo 'mysecrets/' > .yadm/encrypt # Note that 'mysecrets/*' would also fail
yadm encrypt
```

I'm writing a simple patch/pull request to use `ls -d` instead of plain `ls`. This way the folder is not recursed but the entire directory structure is correctly placed in the encrypted tar file.
I believe this doesn't break anything as it treats regular files as before; I tried with several tests, but I can never be 100% sure, so please review.

**Comment by TheLocehiliosan** [comment]
Thanks for the contribution.

It doesn't break any of the current test cases. I'll write a test case for this specifically, and plan to include it in 1.12.0.

**Comment by TheLocehiliosan** [comment]
This has been released in 1.12.0.
